app/api/messages/ping.rb

```diff
@@ -0,0 +1,8 @@
+module Messages
+  class Ping < Grape::API
+    desc 'Returns pong.'
+    get :ping do
+      { ping: params[:pong] || 'pong' }
+    end
+  end
+end
```

Comment:
I think we could remove this endpoint, or at least leave it outside the `Messages` namespace so we can use it as a health check endpoint. What do you think?

Reply:
Done!